src/components/LinkAsButton/test/__snapshots__/index.test.tsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`components/LinkToNewPlans Matches snapshot 1`] = `
 <LinkAsButton
   classNameProp="focus-investigation btn btn-primary float-right mt-5"
-  text="Add Focus Investigation"
+  text="Add Plan"
```

Comment:
@cKellyDesign should we expect this change and the one below?

Reply:
I think probably depends on the context the LinkAsButton is embedded in, checking now

Reply:
It seems there are a bunch of places where it changed like this.

Reply:
Yeah, I think I will change it back and open a ticket to update this b/c it feels like the default props for such a generic component should actually be more generic than `Add Plan` and shouldn't be breaking other, more specific tests where the component is being used.

Good catch, thanks @moshthepitt!

Reply:
Issue created at https://github.com/onaio/reveal-frontend/issues/634